apps/server/src/modules/learnroom/controller/course.controller.spec.ts

```diff
@@ -8,7 +8,8 @@ import { ConfigService } from '@nestjs/config';
 import { CourseExportUc } from '@src/modules/learnroom/uc/course-export.uc';
 import { CourseController } from './course.controller';
 import { CourseUc } from '../uc/course.uc';
-import { CourseUrlParams } from './dto';
+import { CourseUrlParams, CourseQueryParams } from './dto';
+import { CommonCartridgeVersion } from '../common-cartridge';
 
 describe('CourseController', () => {
```

Comment:
Controller should be tested by API tests instead of unit tests. We have to remove the unit tests regarding course export and achieve full coverage with API tests.

Reply:
should we create a separate ticket for removing unit tests of the controller and add new API tests or do you mean we can add it now as a part of the current ticket?

Reply:
As part of the current ticket. We are touching the endpoint therefore we need the proper tests. Sorry, for the additional effort.